### ISSUE TITLE
detect file structure

### ISSUE THREAD

**Issue by AlexisEidelman** [SGMAP-AGD/csv_detective#6]
On peut regarder si on a des variables nommées name1, name2, name3, etc.
On aura des idées en regardant d'autres fichiers

**Comment by Leobouloc** [comment]
Je ne suis pas sûr d'avoir bien compris. Tu veux analyser par le contenu; récupérer tous les noms de labels associés à un sujet; puis réutiliser ces dénominations pour l'analyse de futurs fichiers ? c'est bien ça ?

**Comment by AlexisEidelman** [comment]
Oui, c'est peut-être pas du csv_detective brut mais je voudrais faire un outil, pour passer d'un forme comme ça à une autre. Voir http://www.jstatsoft.org/v59/i10/paper .

La solution est peut-être aussi simple que d'intaller R et le package qui va bien https://ramnathv.github.io/pycon2014-r/explore/tidy.html
ou bien de le réécrire (si ça n'a pas déjà été fait)

**Comment by AlexisEidelman** [comment]
Indice : 
http://nbviewer.ipython.org/github/ResearchComputing/Meetup-Fall-2013/blob/master/python/lecture_21_pandas_processing.ipynb
